modules/cdp-engine/src/lib.rs

```diff
@@ -490,6 +524,26 @@ pub mod module {
 			CollateralParams::<T>::insert(currency_id, collateral_params);
 			Ok(())
 		}
+
+		#[pallet::weight(<T as Config>::WeightInfo::register_liquidation_contract())]
+		#[transactional]
+		pub fn register_liquidation_contract(origin: OriginFor<T>, address: EvmAddress) -> DispatchResult {
+			T::UpdateOrigin::ensure_origin(origin)?;
```

Comment:
we want a different origin to register this. the current UpdateOrigin is financial council but register liquidation contracts should be GeneralCouncil